src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml

```diff
@@ -68,42 +101,81 @@ limitations under the License.
+    <attribute side="server" code="0x000C" define="SUPPORTS_DNS_RESOLVE" type="boolean" optional="true">SupportsDNSResolve</attribute>
 
-    <!-- Test Commands -->
     <command source="client" code="0x00" name="SetUtcTime" optional="false">
-	<description>Upon receipt of this command, the server MAY update its UTCTime Attribute to match the time specified in the command</description>
-	<arg name="UtcTime" type="epoch_us"/>	
-	<arg name="Granularity" type="GranularityEnum"/>	
-	<arg name="TimeSource" type="TimeSourceEnum" optional="true"/>
-    </command>	
+      <description>This command MAY be issued by Administrator to set the time.</description>
+      <arg name="UtcTime" type="epoch_us"/>
+      <arg name="Granularity" type="GranularityEnum"/>
+      <arg name="TimeSource" type="TimeSourceEnum" optional="true"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x01" name="SetTrustedTimeSource" isFabricScoped="true" optional="false">
+      <description>This command SHALL set TrustedTimeSource.</description>
+      <arg name="TrustedTimeSource" type="FabricScopedTrustedTimeSourceStruct" isNullable="true"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x02" name="SetTimeZone" response="SetTimeZoneResponse" optional="true">
+      <description>This command SHALL set TimeZone.</description>
+      <arg name="TimeZone" type="TimeZoneStruct" array="true"/>
+      <access op="invoke" privilege="manage"/>
+    </command>
+
+    <command source="server" code="0x03" name="SetTimeZoneResponse" optional="true">
+      <description>Response to SetTimeZone.</description>
+      <arg name="DSTOffsetRequired" type="boolean"/>
+    </command>
+
+    <command source="client" code="0x04" name="SetDSTOffset" optional="true">
+      <description>This command SHALL set DSTOffset.</description>
+      <arg name="DSTOffset" type="DSTOffsetStruct" array="true"/>
+      <access op="invoke" privilege="manage"/>
+    </command>
+
+    <command source="client" code="0x05" name="SetDefaultNTP" optional="true">
+      <description>This command is used to set DefaultNTP.</description>
+      <arg name="DefaultNTP" type="CHAR_STRING" length="128" isNullable="true"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <event code="0x0000" name="DSTTableEmpty" priority="info" side="server" optional="true">
+      <description>This event SHALL be generated when the server stops applying the current DSTOffset and there are no entries in the list with a larger ValidStarting time.</description>
+    </event>
+
+    <event code="0x0001" name="DSTStatus" priority="info" side="server" optional="true">
+      <description>This event SHALL be generated when the server starts or stops applying a DST offset.</description>
+      <field id="0" name="DSTOffsetActive" type="boolean" />
```

Comment:
The spec does not define this field in a sane way.  It has prose, but not an actual specification.  That needs to exist.  Do you want to file the spec issue, or should I?

Reply:
Agree, https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6573